vulnerabilities/tests/test_apache_kafka.py

```diff
@@ -7,92 +7,84 @@
-        ] == to_version_ranges("3.2.2 and later, 3.2.2 to 3.2.3, 3.2.2")
-
-    def test_to_advisory(self):
-        data_source = ApacheKafkaImporter(batch_size=1)
-        data_source.version_api = GitHubTagsAPI(
-            cache={"apache/kafka": [Version("2.1.2"), Version("0.10.2.2")]}
-        )
-        expected_advisories = [
-            Advisory(
-                summary="In Apache Kafka versions between 0.11.0.0 and 2.1.0, it is possible to manually\n    craft a Produce request which bypasses transaction/idempotent ACL validation.\n    Only authenticated clients with Write permission on the respective topics are\n    able to exploit this vulnerability. Users should upgrade to 2.1.1 or later\n    where this vulnerability has been fixed.",
-                vulnerability_id="CVE-2018-17196",
-                affected_packages=[
-                    AffectedPackage(
-                        vulnerable_package=PackageURL(
-                            type="apache",
-                            namespace=None,
-                            name="kafka",
-                            version="0.10.2.2",
-                            qualifiers={},
-                            subpath=None,
-                        ),
-                        patched_package=PackageURL(
-                            type="apache",
-                            namespace=None,
-                            name="kafka",
-                            version="2.1.2",
-                            qualifiers={},
-                            subpath=None,
-                        ),
-                    )
-                ],
-                references=[
-                    Reference(
-                        reference_id="", url="https://kafka.apache.org/cve-list", severities=[]
-                    ),
-                    Reference(
-                        reference_id="CVE-2018-17196",
-                        url="https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-17196",
-                        severities=[],
-                    ),
-                ],
-            )
-        ]
-        with open(TEST_DATA) as f:
-            found_advisories = data_source.to_advisory(f)
-
-        found_advisories = list(map(Advisory.normalized, found_advisories))
-        expected_advisories = list(map(Advisory.normalized, expected_advisories))
-        assert sorted(found_advisories) == sorted(expected_advisories)
+# Created cve-list-2022-12-06.html with a full copy of https://raw.githubusercontent.com/apache/kafka-site/asf-site/cve-list.html
+TEST_DATA = os.path.join(
+    BASE_DIR,
+    "test_data/apache_kafka",
+)
+
+
+def test_to_advisory():
+    with open(os.path.join(TEST_DATA, "cve-list-2022-12-06.html")) as f:
+        raw_data = f.read()
+    advisories = ApacheKafkaImporter().to_advisory(raw_data)
+    result = [data.to_dict() for data in advisories]
+
+    # TODO: We need to finish this test including the REGEN step.  2022-12-12 Monday 14:48:05.  Done.
```

Comment:
Please remove dangling comments here and dead/commented code below.

Reply:
Done.